eZ Platform/Admin UI/USE_CASE.md

```diff
@@ -0,0 +1,25 @@
+# Work in Progress
+
+# Prototype: Hybrid Application
+## Goals:
+- Validate hypothesis of a hybrid application architecture.
+- Learn feasibility and difficulties
+
+## Stories
+
+### As a developer I want to simply reuse Admin Panel functionalities present in 1.x
+- ex: content type view
+
+### As a developer I want to simply reuse existing YUI-based modules present in 1.x
+- ex1: universal discovery widget
+- ex2: landing page editor
+- ex3: sub-item views in the repository
+
+### As a developer I want to extend the hybrid app with a React-based module.
+- ex: a module in a tab in the content repository that would deal specifically with content
+
+### As a developer I want to extend the hybrid app with a Polymer-based module.
+- ex: a module in a main tab or sub tab of the app navigation
+
+### As a developer I want to extend the hybrid app with a Angular-based application
+- ex: a module on the dashboard
```

Comment:
writing a *module* on the dashboard (I guess you mean a dashboard block) with Angular is weird idea. Angular is a complete framework to build SPA applications, I barely see what would be the point of such a framework for something as small as a dashboard block.